src/layer/image/index.ts

```diff
@@ -106,7 +108,7 @@ const SHADER_JSON_KEY = "shader";
 const SHADER_CONTROLS_JSON_KEY = "shaderControls";
 const CROSS_SECTION_RENDER_SCALE_JSON_KEY = "crossSectionRenderScale";
 const CHANNEL_DIMENSIONS_JSON_KEY = "channelDimensions";
-const VOLUME_RENDERING_JSON_KEY = "volumeRendering";
+const VOLUME_RENDERING_MODE_JSON_KEY = "volumeRenderingMode";
```

Comment:
While the existing volume rendering support was marked experimental, we should still retain the existing json key (and handle true/false) to avoid breaking compatibility with existing states (and urls).